modules/social_features/social_book/social_book.install

```diff
@@ -241,3 +242,17 @@ function social_book_update_8902() {
   // Make sure we clear cached definitions for the fields.
   \Drupal::service('entity_field.manager')->clearCachedFieldDefinitions();
 }
+
+/**
+ * Add small_teaser view mode for book.
+ */
+function social_book_update_8903() {
+  $config_path = drupal_get_path('module', 'social_book') . '/config/optional';
```

Comment:
Please use a static file for this, config/optional can change when we export in the future and than our updates might fail.

Reply:
I see it was fixed.